tests/lib/utils/import-test.js

```diff
@@ -21,3 +21,45 @@ describe('getSourceModuleName', () => {
     expect(importUtils.getSourceModuleName(node)).toStrictEqual('Model');
   });
 });
+
+describe('getImportIdentifier', () => {
+  it('gets an empty array when no import is found', () => {
+    const node = babelEslint.parse("import { later } from '@ember/runloop';").body[0];
+    expect(importUtils.getImportIdentifier(node, '@ember/object', 'action')).toStrictEqual('');
+  });
+
+  it('gets an array of identifiers when found', () => {
+    const node = babelEslint.parse("import { later } from '@ember/runloop';").body[0];
+    const identifier = importUtils.getImportIdentifier(node, '@ember/runloop', 'later');
+
+    expect(identifier).toStrictEqual('later');
+  });
+
+  it('gets an aliased identifier when found', () => {
+    const node = babelEslint.parse("import { later as laterz } from '@ember/runloop';").body[0];
+    const identifier = importUtils.getImportIdentifier(node, '@ember/runloop', 'later');
+
+    expect(identifier).toStrictEqual('laterz');
+  });
+
+  it('gets an identifier when found for default imports', () => {
+    const node = babelEslint.parse("import Component from '@ember/component';").body[0];
+    const identifier = importUtils.getImportIdentifier(node, '@ember/component', 'Component');
```

Comment:
This should not have a 3rd parameter passed since it's just looking for the default import.

Reply:
See above comment.